pom.xml

```diff
@@ -1168,12 +1168,14 @@ under the License.
         <plugin>
           <groupId>com.diffplug.spotless</groupId>
           <artifactId>spotless-maven-plugin</artifactId>
-          <version>2.37.0</version>
+          <version>2.39.0</version>
           <configuration>
             <java>
               <!-- orders of used formatters are important MPOM-376 -->
               <!-- eg. palantir override importOrder, so should be first -->
-              <palantirJavaFormat />
+              <palantirJavaFormat>
+                <version>2.36.0</version>
+              </palantirJavaFormat>
```

Comment:
please add link to spotless issue https://github.com/diffplug/spotless/issues/1774